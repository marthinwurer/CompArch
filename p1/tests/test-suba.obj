50 1 00001
51 1 00005
52 1 fffff
53 1 ffffb
100 1 10050
101 1 22050
102 1 10050
103 1 22051
104 1 22050
105 1 10050
106 1 22052
107 1 10050
108 1 22053
109 1 00000
100
