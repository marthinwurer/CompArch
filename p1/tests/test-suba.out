CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 100
100:  10050  LDA   050  A[00000]  B[00000]  XR[000]
101:  22050  SUBA  050  A[00001]  B[00000]  XR[000]
102:  10050  LDA   050  A[00000]  B[00000]  XR[000]
103:  22051  SUBA  051  A[00001]  B[00000]  XR[000]
104:  22050  SUBA  050  A[ffffc]  B[00000]  XR[000]
105:  10050  LDA   050  A[ffffb]  B[00000]  XR[000]
106:  22052  SUBA  052  A[00001]  B[00000]  XR[000]
107:  10050  LDA   050  A[00002]  B[00000]  XR[000]
108:  22053  SUBA  053  A[00001]  B[00000]  XR[000]
109:  00000  HALT       A[00006]  B[00000]  XR[000]

Machine Halted - HALT instruction executed

Simulated time 53 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
